lib/api/v1/forms/sign_up.go

```diff
@@ -0,0 +1,46 @@
+package forms
+
+import (
+	"github.com/markgravity/golang-ic/database"
+	"github.com/markgravity/golang-ic/helpers"
+	"github.com/markgravity/golang-ic/helpers/log"
+	"github.com/markgravity/golang-ic/lib/models"
+	"github.com/markgravity/golang-ic/lib/validators"
+)
+
+type SignUpForm struct {
+	Email    string `form:"email" binding:"required,email"`
+	Password string `form:"password" binding:"required,min=6"`
+}
+
+func (f *SignUpForm) Validate() error {
+	return validators.Validate(f)
```

Comment:
I think we don't need to call this. It will automatically call when you use `context.ShouldBindQuery`. Please check https://github.com/nimblehq/gulf-approval-web as well